src/main/markdown/gettingstarted-outdated.md

```diff
@@ -0,0 +1,162 @@
+Getting Started
+===
+
+Note: Starting with GWT 2.11.0 the webAppCreator is deprecated. New instructions located [here](gettingstarted.hmtl). 
```

Comment:
Try to avoid "click here" sorts of links, instead make the relevant text _be_ the link (also, typo in url):
```suggestion
Note: Starting with GWT 2.11.0 the webAppCreator is deprecated, please see the [new instructions](gettingstarted.html). 
```

Obviously not needed if we delete this old file.